### DISCUSSION TITLE
can 11ty be hosted on Hostinger?

### DISCUSSION THREAD

**Discussion by ar-elsaid** [11ty/eleventy#2398] [Q&A]
recently I got a very good deal on Hostinger shared hosting.

I'm new to the JAMStack architecture.

since 11ty is JavaScript based:

1# can I build my website using 11ty and then host it on Hostinger?

2# will I face any problems onwards if I used Hostinger as my primary hosting solution with 11ty?

**Comment by prtksxna** [comment]
Services like Hostinger are often optimized for CMSs like WordPress. But, since they provide a way to deploy a git repository you could try going that way. See their documentation here https://support.hostinger.com/en/articles/1583302-how-can-i-deploy-a-git-repository
